org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java

```diff
@@ -30,6 +30,7 @@
 	private static final String SPACE_BEFORE_EMPTY_CLOSE_TAG = "spaceBeforeEmptyCloseTag";
 	private static final String QUOTATIONS = "quotations";
 	private static final String JOIN_CONTENT_LINES = "joinContentLines";
+	private static final String PRESERVE_NEWLINES = "preserveNewlines";
```

Comment:
now that it's a number, it makes more sense to rename it to PRESERVED_NEWLINES="preservedNewLines"